src/components/TodoList.js

```diff
@@ -0,0 +1,57 @@
+import React, {Component} from 'react';
+import TodoItem from './TodoItem';
+
+export default class TodoList extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      requested: false,
+      loaded: false,
+      items: []
+    }
+    this.getItem = this.getItem.bind(this);
+    this.sortItems = this.sortItems.bind(this);
+  }
+
+   loadingUrl(url) {
+    return fetch(url)
+    .then(response => response.json()).then(response => response);
+  }
+
+  getItem() {
+    this.setState({
+      requested: true
+    });
+    Promise.all([this.loadingUrl(' https://jsonplaceholder.typicode.com/todos'), 
+      this.loadingUrl('https://jsonplaceholder.typicode.com/users')])
+      .then(([todos, users]) => this.setState({
+        loaded: true,
+        items: todos.map(item => ({
+          ...item,
+          user: users.find((i) => i.id === item.userId)
+        }))
+      }));
+  }
+
+  sortItems() {
+    this.setState(prevState => ({
```

Comment:
Additionally, you should provide a capability of sorting the items either by title, user or its status (whether the item is completed or not).